pulp_2to3_migration/app/plugin/content.py

```diff
@@ -310,16 +312,31 @@ def get_remote_by_importer_id(importer_id):
 
                         lces = pulp2lazycatalog.filter(pulp2_storage_path=image_path)
                         if lces:
+                            remote_declarative_artifacts = []
+
                             for lce in lces:
                                 remote = get_remote_by_importer_id(lce.pulp2_importer_id)
+
+                                if not remote and not downloaded:
+                                    continue
+
                                 remotes.add(remote)
```

Comment:
Maybe this should be under `if remote` condition, to cover the case when a remote is not present but the item is downloaded. What do you think? to avoid have a set with None value in it and create Declarative artifact with it later.

Reply:
Agree